ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -97,6 +98,17 @@ function TransactionListItemInner({
       isEarliestNonce,
     });
 
+  const getTestNetworkBackgroundColor = (networkId) => {
+    switch (true) {
```

Comment:
Should this be switching on `networkId` instead? Also, I wonder if this function can live outside of this component so that it doesn't de-reference. Maybe we can at least wrap it in a `useCallback` hook.